app/src/main/java/com/codepath/apps/restclienttemplate/DetailActivity.java

```diff
@@ -0,0 +1,102 @@
+package com.codepath.apps.restclienttemplate;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.FragmentManager;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.bitmap.CircleCrop;
+import com.codepath.apps.restclienttemplate.databinding.ActivityDetailBinding;
+import com.codepath.apps.restclienttemplate.models.Tweet;
+
+import org.parceler.Parcels;
+
+public class DetailActivity extends AppCompatActivity {
+
+    Tweet tweet;
+
+    ImageView ivProfileImage;
+    ImageView ivMedia;
+    ImageView ivReply;
+    ImageView ivRetweet;
+    ImageView ivFavorite;
+    TextView tvBody;
+    TextView tvScreenName;
+    TextView tvName;
+    TextView tvTime;
+    TextView tvRetweet;
+    TextView tvFavorite;
+
+    ActivityDetailBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        //setContentView(R.layout.activity_detail);
```

Comment:
Remove those if you've already submitted this code, you won't use it anymore yet it can confuse reader